docs/dev_guide/contents/pr_review_procedure.rst

```diff
@@ -25,11 +25,30 @@ Before the "merge" button is clicked the following criteria must be met:
 
 * All the continuous integration must pass unless there is a known issue.
 
-* At least two members (not the author of the PR) of the "sunpy-developers" group must have approved the PR, one should, ideally, be a relevant subpackage maintainer.
+* At least two members (excluding the PR author) of the "sunpy-developers" group must have approved the PR.
```

Comment:
This is an aside for this PR, but this would become project members in future?

Reply:
My understanding wws from the SEP is that the two groups (project members and sunpy maintainers) remain distinct, even though all maintainers are to be made members, members wouldn't be made made maintainers necessarily when voted in. This is probably worth raising on the SEP pr for clarification though.

Reply:
yeah, maintainers of packages aren't the same group as project members. Not all project members will be developers, and certainly not developers of core or all packages etc.